marlowe/src/Language/Marlowe/Client.hs

```diff
@@ -0,0 +1,222 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DefaultSignatures  #-}
+{-# LANGUAGE DeriveAnyClass     #-}
+{-# LANGUAGE DeriveGeneric      #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE RecordWildCards   #-}
+{-# LANGUAGE RankNTypes   #-}
+{-# LANGUAGE NamedFieldPuns   #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE TemplateHaskell   #-}
+{-# OPTIONS -fplugin=Language.PlutusTx.Plugin
```

Comment:
You shouldn't need this any more - if you get errors without this now they're bugs.

Reply:
I do get errors without this. I'll create an issue.

Reply:
Fixed, no need for plugin.